### DISCUSSION TITLE
Bug when importing graph data with the Variable Importer ?

### DISCUSSION THREAD

**Discussion by StevenRiot** [Silly-V/Adobe-Illustrator#70] [Q&A] [answered]
Am I the only one with graph data not being imported properly (text variables are fine). 

I get prompted to “Cycle Update All Datasets” but it's stuck on the first item and won't go further.
If I close the modal, all my graphs have a '1.00' in the first cell and nothing else.

Has it happened to anyone else ? Any ideas on how to fix it ?

**Comment by Silly-V** [comment]
How'd you fix it?

**Reply by StevenRiot** [reply, marked as answer]
It was actually a problem with the path to graph's data files. I changed their location to another folder and it worked. Might be nice to add an error when this happens.
